transform.ts

```diff
@@ -241,6 +240,25 @@ function createInterface(path: NodePath, componentTypes: CollectedTypes) {
 
   return typeName
 }
+/**
+ * If forwardRef is being used, declare the props.
+ * Otherwise, return false
+ */
+function addForwardRefTypes(path: NodePath, typeName: string): boolean {
+  // for `React.forwardRef()`
+  let isForwardRef = path.node.callee?.property?.name === "forwardRef"
+  if (isForwardRef) {
+    path.node.callee.property.name = `forwardRef<React.ReactHTMLElement, ${typeName}>`
+    return isForwardRef
+  }
+  // if calling `forwardRef()` directly
+  isForwardRef = path.node.callee?.name === "forwardRef"
```

Comment:
Let's cleanup this duplication by making a function that finds the forward ref node, then we can act on that node to add the types.

Reply:
this is better!